style.css

```diff
@@ -542,34 +542,125 @@ nav {
   transform: scale(1.1);
 }
 
-button:hover {
-  cursor: pointer;
-  color: chocolate;
+footer {
+  position: absolute;
+  top: 5990px;
+  left: 0;
+  display: flex;
+  flex-direction: column;
+  width: 375px;
+  height: 950px;
 }
 
-input[type="submit"]:hover {
-  cursor: pointer;
-  color: chocolate;
+.form-container {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 120px 24px;
+  gap: 58px;
+  width: 375px;
+  height: 844px;
 }
 
-button:active {
-  background-color: chocolate;
-  border: 2px solid chocolate;
-  color: white;
+.form-text-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 4px;
+  gap: 24px;
+  width: 327px;
+  height: 216px;
 }
 
-input[type="submit"]:active {
-  background-color: chocolate;
-  border: 2px solid chocolate;
-  color: white;
+.form-text-container h2 {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 1200;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  color: #172b4d;
 }
 
-button:disabled {
-  background-color: #ccc;
-  color: gray;
+form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0;
+  gap: 24px;
+  width: 327px;
+  height: 330px;
+}
+
+input {
+  width: 327px;
+  height: 48px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  color: #979493;
+  border-top: none;
+  border-left: none;
+  border-right: none;
+  border-bottom: 1px solid #dbd8d7;
+  padding: 10px;
+}
+
+textarea {
+  width: 327px;
+  height: 114px;
+  background-color: #fbf8f7;
+  border: none;
+  padding: 10px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+}
+
+textarea::placeholder {
+  color: #3c3a39;
+}
+
+.btn-form {
+  padding: 12px;
+  gap: 10px;
+  width: 132px;
+  height: 48px;
+  border: none;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+}
+
+.bottom_navigation {
+  width: 375px;
+  height: 82px;
+  border-top: 1px solid #dfe1e6;
+  display: flex;
+  align-items: center;
+  justify-content: center;
 }
 
-input[type="submit"]:disabled {
+button:hover {
+  cursor: pointer;
+  color: brown;
+}
```

Comment:
- Please implement the button interactions for your form button (I'd recommend you do these for all other buttons also) 🙏 . Please remember replicating exact Figma templates gives confidence to recruiters in your ability to replicate business requirements as regards design 😄 

Figma template (button interactions to be implemented, pay attention to the bg color, text color, and border colors):
![button interact](https://user-images.githubusercontent.com/73607512/192638750-04c9ab1c-475a-4eb8-b129-10f0701931f3.PNG)

Reply:
Hi @mckent05,

Yes I have now implemented all of these designs according to Figma but I think their behavior is hidden due to button-enabled implementation